### ISSUE TITLE
Server MineOS fails to install with GUI error

### ISSUE THREAD

**Issue by nitreojs** [IgorTimofeev/MineOS#369]
Everything is the same as https://github.com/IgorTimofeev/MineOS/issues/171, except we get kinda different error from that issue.
Note that we did enable `lua: allowBytecode=true`.

```
machine:1505: bios:26: installer:197: load =GUI: java.lang.NullPointerException
stack traceback:
  [Java]: in ?
```
![image](https://user-images.githubusercontent.com/40319427/125445382-1db96b33-a6ca-45fb-8e8a-bec3e9a98567.png)

**Comment by Sosiskavteste28** [comment]
If you play minecraft on android, most likely forget about Mine OS. This is most likely due to LuaJ, the only solution is to make the server using Aternos or similar and install the system there

**Comment by MrFredber** [comment]
@Sosiskavteste28 how you gonna play on Android with minecraft JAVA? 
@nitreojs try to reinstall java to a newer version or install java x64 if your pc is x64

**Comment by CoolCat467** [comment]
@Sosiskavteste28 Minecraft on Android is Bedrock Edition. From my knowledge, mods like OpenComputers aren't even supported in Bedrock Edition. 
@nitreojs MrFredber is correct, attempt re-installing java, but also try reinstalling OpenComputers itself. From the details given, this does not appear to be an issue with MineOS, but OpenComuters.

**Comment by Sosiskavteste28** [comment]
> @Sosiskavteste28 Minecraft on Android is Bedrock Edition. From my knowledge, mods like OpenComputers aren't even supported in Bedrock Edition.
I play minecraft from my phone through PojavLauncher

**Comment by Bioscreeper** [comment]
It might be arm - maybe it only works on x86/x64? I tried on a pi 4, did'nt work (same error)

**Comment by MrFredber** [comment]
raspberry pi 4 built on arm architecture?

**Comment by Bioscreeper** [comment]
yes

**Comment by Bioscreeper** [comment]
pi IS arm

**Comment by CoolCat467** [comment]
A raspberry pi computer uses an arm processor. Raspberry pi 0 w, 3, and 4 all support using a 64 bit operating system, but most commonly only have a 32 bit OS installed. Minecraft on a raspberry pi 4 with a 32 bit os works, but though use on a 64 bit os is theoretically possible, I have not been able to do it myself. Now, all of this is unrealated though, as MineOS, the OpenComputers Operating System for Minecraft (not the minecraft server os thing, that's a different thing all together and you're in the wrong place if that's what you're here for) shouldn't care how many bits your OS uses or what computer architecture it's processor is. That's Java's problem. Minecraft itself is running on top of java, so it doesn't care about all that.

**Comment by TheSainEyereg** [comment]
This is due to the fact that the Open Computers mod does not have native lua libraries for the ARM architecture. I [described this issue](https://github.com/MightyPirates/OpenComputers/issues/3466) in the main OpenComputers repository. You can do the same as I did for my Oracle servers, just put the libraries directly into the JAR file and specify the force using this library in OpenComputers config.

**Comment by gatoborrachon** [comment]
> I play minecraft from my phone through PojavLauncher

@BNr-SP did you found a way to avoid computers rebooting everytime you enter to a world?

**Comment by Pro12447890** [comment]
Similar problem. As with many other users of these operating systems. It worked calmly on Aternos, and, if I’m not mistaken, in single-player too. But, since last year, it is now impossible to boot BIOS and OpenOS at all, and the drives are also affected by the blue display of death.

**Comment by MrFredber** [comment]
> As with many other users of these operating systems. ...  it is now impossible to boot BIOS and OpenOS at all, and the drives are also affected by the blue display of death.

I disagree with you. Everything is working perfectly. MineOS, OpenOS and FOS (my OS) is working totally normal in singleplayer and multiplayer more than 5 years on many pc's (Windows, MacOS, Aternos etc)
Has your mod version been updated to 1.8.7?

**Comment by MrFredber** [comment]
![Image](https://github.com/user-attachments/assets/a015a085-4eb4-4101-983a-c5ca1c48ae5f) Freshly installed MineOS right now

**Comment by Smok1e** [comment]
> Similar problem. As with many other users of these operating systems. It worked calmly on Aternos, and, if I’m not mistaken, in single-player too. But, since last year, it is now impossible to boot BIOS and OpenOS at all, and the drives are also affected by the blue display of death.

If you are getting the same error message as the OP does, it is most likely due to a lack of lua natives for arm architecture in OpenComputers until [1.8.0](https://github.com/MightyPirates/OpenComputers/releases/tag/1.12.2-forge%2F1.8.0) was released. If so, consider updating OC to the latest version.

Otherwise, please clarify what problem you are facing with.
